core/src/main/java/tc/oc/pgm/modules/ModifyBowProjectileMatchModule.java

```diff
@@ -123,6 +127,16 @@ public void fixEntityDamage(EntityDamageByEntityEvent event) {
               velocity.getZ() + projectileVelocity.getZ() * knockback * 0.6 / horizontalSpeed);
           event.getEntity().setVelocity(velocity);
         }
+
+        // If the projectile is not an arrow, play an impact sound.
+        if (event.getEntity() instanceof Player
+            && (projectile instanceof Projectile && !(projectile instanceof Arrow))) {
+          Projectile customProjectile = (Projectile) projectile;
+          if (customProjectile.getShooter() instanceof Player) {
+            Player bukkitShooter = (Player) customProjectile.getShooter();
+            SoundsMatchModule.playSound(match.getPlayer(bukkitShooter), PROJECTILE_SOUND);
```

Comment:
I am not convinced that this is an optional sound. Vanilla minecraft does the ding no matter your sound setting, and this also should

Reply:
I would agree, let's keep it default.

Reply:
@Pablete1234 and @Electroid thanks for clarifying this. I was unsure about whether to make it default or not and I agree, it’s been adjusted!